node/src/daemon/mod.rs

```diff
@@ -258,14 +261,16 @@ impl Daemon {
                         },
                         payload: match body.payload {
                             Ok(json) => json,
-                            Err((_code, _message)) => unimplemented!(),
+                            Err((_code, _message)) => {
+                                panic!("Incoming message from UI with a payload signaling an error")
```

Comment:
A bad message from a UI shouldn't crash the Daemon. Put unimplemented!() back here (or maybe the new todo!() if you like that better) and we'll make a card to determine what should be done here.